### ISSUE TITLE
CSS variables not work

### ISSUE THREAD

**Issue by markox92** [jaraco/cssutils#55]
Why does the variable not work?

```
:root{
    --font:"trebuchet ms";
    --primarybg: #000000
}
```

Output
```
WARNING Property: Unknown Property name. [328:5: --font]
Property: Unknown Property name. [328:5: --font]
```

**Comment by jaraco** [comment]
Thanks for the report. I (via Copilot) investigated this, and the warning appears to come from a limitation in `cssutils`, not from invalid CSS on your side.

Your example uses **CSS custom properties**:

```css
:root {
    --font: "trebuchet ms";
    --primarybg: #000000;
}
```

Those `--*` names are modern CSS variable syntax, but `cssutils` currently validates property names against its known property profiles and does not recognize custom properties as valid property names.

In particular, in `cssutils/css/property.py`, property names are checked in `_setName`, and anything not present in `cssutils.profile.knownNames` is logged as an unknown property:

```python name=cssutils/css/property.py url=https://github.com/jaraco/cssutils/blob/304f1c0b270dcd2860c9abf41cf58f8ce33e1f9c/cssutils/css/property.py#L180-L197
if self._isValidating() and self._name not in cssutils.profile.knownNames:
    self._log.warn(
        'Property: Unknown Property name.', token=token, neverraise=True
    )
```

There is also a comment in `validate()` that suggests CSS variable support was anticipated but not actually completed:

```python name=cssutils/css/property.py url=https://github.com/jaraco/cssutils/blob/304f1c0b270dcd2860c9abf41cf58f8ce33e1f9c/cssutils/css/property.py#L388-L406
if self.name and self.value:
    # TODO
    # cv = self.propertyValue
    # if cv.cssValueType == cv.CSS_VARIABLE and not cv.value:
    #     # TODO: false alarms too!
    #     cssutils.log.warn(u'No value for variable "%s" found, keeping '
    #                       u'variable.' % cv.name, neverraise=True)
```

So the behavior is:

- the declaration is parsed syntactically,
- but during validation, `--font` and `--primarybg` are treated as unknown property names,
- which produces the warning you saw.

In short: **your CSS is valid modern CSS, but `cssutils` does not currently have proper support for CSS custom properties/variables**.

As a minor aside, your sample is also missing a trailing semicolon after `--primarybg: #000000`, though that is separate from the `Unknown Property name` warning.

So I believe this issue should be treated as a **missing feature / unsupported modern CSS syntax** in `cssutils`, rather than user error.

**Comment by jaraco** [comment]
Feel free to review the open issues or open one if it doesn't capture the feature(s) you desire.
